cvat/settings/base.py

```diff
@@ -503,6 +515,8 @@ class CVAT_QUEUES(Enum):
     },
 }
 
+LOG_IMPORT_ERRORS = True
```

Comment:
I would suggest starting all cvat related options with the CVAT_ prefix. Also, please add support for configuring this via an environment variable, it's much easier in some cases